### PR TITLE
Hover Color feature supports scrolling, non-100% OS desktop scaling, and other improvements

### PR DESCRIPTION
**Pull request by cmorley191**
Improved hover color indicator appearance.
Also now supports scrolling, has a loading gif while the screenshot is being taken, and is a little more reliable.

### CODE REVIEW

**Comment by cmorley191** [comment]
Also supports chrome zooming!

**Comment by cmorley191** [comment]
Gif of feature:
hoverscroll

File: src/background/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import {Extension} from './extension';
 import {getHelpURL} from '../utils/links';
+import CustomSettingsToggle from 'ui/popup/components/custom-settings-toggle';
```
Comment (cmorley191) [inline src/background/index.ts]: Unneeded import
```suggestion
```

File: src/ui/popup/components/cb-settings/hoverFunctions.js [hunk 2]
```diff
@@ -87,14 +14,38 @@ Things Checked:
     onmousemove statements in a try catch block to deal with these exceptions.
 */
 export function hoverFunVer2(){
```
Comment (cmorley191) [inline src/ui/popup/components/cb-settings/hoverFunctions.js:16]: Should have a better function name

**Review by agatt1 — APPROVED** [review]
Works great

**Pull request merged**